lib/scripts/game_script.rb

```diff
@@ -16,16 +16,29 @@ def initialize(colorizer)
     @errors = {
       mode: colorizer.magenta("\nNot a valid selection. Pick again.\n"),
       order: colorizer.magenta("\nNot a valid selection. Pick again.\n"),
-      token: colorizer.magenta("\nPlease enter a single letter which hasn't already been selected.\n"),
+      token: colorizer.magenta("\nPlease select a valid token from the list below.\n"),
       board: colorizer.magenta("\nNot a valid selection. Pick again.\n"),
       invalid_spot: colorizer.magenta("\nYour selection is not a valid spot. Pick again.\n"),
       out_of_range: colorizer.magenta("\nYour selection is not between 1 and 9. Pick again.\n"),
       spot_taken: colorizer.magenta("\nThat spot has already been selected. Pick again.\n")
     }
   end
 
-  def create_token_prompt(player)
-    "#{@token_selection}#{player}: "
+  def create_token_prompt(emojis, player)
+    output = "#{@token_selection}#{player}.\n\n     "
+
+    emojis.each_with_index do |code, code_index|
+      output += "#{colorizer.green("[#{code_index + 1}]".ljust(5, ' '))} #{code}"
+      if (code_index != emojis.length - 1)
+        output+= ' , '
+      end
+
+      if ((code_index + 1) != emojis.length && (code_index + 1) % 10 == 0)
```

Comment:
what does this do?

Reply:
Pulled this into a [function](https://github.com/saralein/tic-tac-toe_ruby/pull/22/files#diff-818a75ea9e19d71dfb4d1f5419d9d436R43) which I hope will better indicate what it's doing.